messagequeues/ImageTransformationService.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+using RabbitMQ.Client;
+using RabbitMQ.Client.Events;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Tesseract;
+using System.Drawing;
+
+namespace messagequeues;
+
+public class ImageTransformationService
+{
+     public ImageTransformationService()
+     {
+            var ocrengine = new TesseractEngine(@".\tessdata", "eng", EngineMode.Default);
+            var img = Pix.LoadFromFile("test.jpg");
+            var res = ocrengine.Process(img);
+            Console.WriteLine(res.GetText());

```

Comment:
tesseract is not required in scope of the task. This service should listen created queue and save files to a local system